src/main/scala/community/flock/sbt/renderer/ScalaRulesRender.scala

```diff
@@ -8,18 +8,18 @@ import sbt.*
 final class ScalaRulesRender(artifactRef: ArtifactReferenceRenderer) {
 
   def toBuild(module: BuildModule): Starlark = {
-    def buildArgs(name: String, dirType: String, plugins: List[Starlark], deps: List[Starlark]) =
```

Comment:
Had je hier dubbele deps al? Dit is _per_ sbt module, lijkt me gek dat daar al dubbel deps inzitten

Reply:
We schrijven per `bazeModules` sbt Task de deps weg in een file en daarna lezen we alle `bazel/deps` file in in de `bazelDeps` target, wellicht moet je daarvan die _hele_ lijst een set maken?

Reply:
Welke lijst bedoel je dan ?

Reply:
https://github.com/flock-community/sbt-bazel/blob/main/src/main/scala/community/flock/sbt/BazelPlugin.scala#L46

Reply:
Heb hetzelfde nu, gefixt in `main`